src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, useEffect, useState } from 'react';
+
+import './Avatar.scss';
+
+interface AvatarProps {
+  address?: string;
+  className?: string;
+}
+
+const Avatar: FC<AvatarProps> = ({ address = '', className = '' }) => {
+  const [linkIcon, setlinkIcon] = useState<string>('');
+  async function getIcon(): Promise<string> {
+    const response = await fetch(
```

Comment:
This should be done in the store. Make a `user` store in `redux/stores` and save the avatar there. This way we can reuse it in other components and also cache it to localStorage.

If this is too complicated just let me know, I can help you out.